### DISCUSSION TITLE
好きなアルファベット打ってください!

### DISCUSSION THREAD

**Discussion by keurou** [NAIST-SE/AttractingContributors#237] [Ideas]
https://github.com/KentaroUedaNAIST/AttractingContributors

**Comment by h3y6e** [comment]
I added. If there are no problems, please merge.

**Comment by t-matsumoto4758** [comment]
Added!
please check and help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by xincanfeng** [comment]
added!
can you tell my you favorite color?
https://github.com/FengXincan/AttractingContributors-color

**Comment by K-Ito-2424** [comment]
I added! Please help me if you can.
https://github.com/K-Ito-666/etc
